inc/agent.class.php

```diff
@@ -1199,7 +1199,15 @@ protected function enrollByInvitationToken($input) {
       $pfAgent = new PluginFusioninventoryAgent();
       //var_dump($_SESSION['glpi_plugin_fusioninventory']);
       $_SESSION['glpi_fusionionventory_nolock'] = true;
+      ob_start();
       $pfCommunication->handleOCSCommunication('', $inventory, 'glpi');
+      $fiErrors = ob_get_contents();
+      if ($fiErrors) {
+         // FI print errors to sdt output and agents needs a correct response, let's save this to logs.
+         $event = __($fiErrors, 'flyvemdm');
```

Comment:
This is not the responsibility ot Flyve MDM to localize strings generated by Fusion Inventory.

Moreover, this local cannot be detected by gettext then it is source of hassle to localize it. 

I think we should log the text from FI as is.